### ISSUE TITLE
Thank you, best example for ULN2003 I could find

### ISSUE THREAD

**Issue by jamogle** [mperino/CircuitPython-28BYJ-48-ULN2003#1]
Thanks for your example. just a small addition, after reading the release() method that adafruit use for depowering the coils when you're done, you just set them all to 0 

def release():
    pin1.value = 0
    pin2.value = 0
    pin3.value = 0
    pin4.value = 0

**Comment by mperino** [comment]
Yup, I've done that in a few other code snippets with these.  Slowly learning the Pythonic way to do stuff but keeping it simple and understandable are the main goals.  Happy that this helped ya.
